x-pack/plugins/ingest_manager/server/routes/setup/handlers.ts

```diff
@@ -81,6 +82,13 @@ export const ingestManagerSetupHandler: RequestHandler = async (context, request
       body: { isInitialized: true },
     });
   } catch (e) {
+    if (e instanceof IngestManagerError) {
+      logger.error(e.getMessage());
+      return response.customError({
+        statusCode: getHTTPResponseCode(e),
+        body: { message: e.getMessage() },
+      });
+    }
     if (e.isBoom) {
       logger.error(e.output.payload.message);
```

Comment:
This will be removed once we stop using `Boom` elsewhere. This handler is currently in an in-between state where it should capture both `Boom` and `IngestManagerError` errors